index.rst

```diff
@@ -64,17 +54,19 @@ for discussion around a future HTTP client PSR.
 
 .. toctree::
     :hidden:
-    :maxdepth: 3
 
     PHP-HTTP <self>
 
 .. toctree::
    :hidden:
-
-   HTTPlug <httplug>
-
-.. toctree::
-   :hidden:
+   :caption: HTTPlug
```

Comment:
i split out the overview from the introduction to avoid having this label with no link behind it. i would prefer to not have titles in the toc that dont link to anything. and currently httplug chapters are grouped together, this pr puts them on the same level as clients or plugins.